web_common/web_common/styles/main.scss

```diff
@@ -1,68 +1,101 @@
 @import "variables/colors.scss";
 @import "variables/margins.scss";
-@import "buttons.scss";
-@import "typography.scss";
-@import "input.scss";
-@import "spinkit-mod.scss";
 
 html {
-  box-sizing: border-box;
-  font-size: 16px;
+    box-sizing: border-box;
 }
-
-button,
-input {
-  margin: 0px;
-  font-size: 0.875rem;
-}
-
 *,
 *:before,
 *:after {
-  box-sizing: inherit;
+    box-sizing: inherit;
 }
 
 body {
-  margin: 0px;
-}
-
-#theme-site {
```

Comment:
Not sure why we got rid of this

Reply:
I admit I didn't understand the point of this.  Didn't seem necessary to me.

Reply:
The #theme-site was a top-level identifier, to add more specificity for rules (in case of conflict with styling libraries we may in the future use), and to allow the theme to change (click a button, change theme-site to theme-site-dark or whatever). But I was asking about the rules. defined within theme-site, which defined some base styles

Reply:
Ah, I see.  I'm inclined to put it back when I actually need it for an application like you mention.  Changing the id on a container to change the styling is a slick trick.  I'm continually impressed with the stuff you can do that browsers can roll with.

It's all there, the relevant stuff anyway.  I added a `body { ... }` run that has the margin and font stuff.  The layout stuff didn't need to go anywhere, I'm using the default font-size (to be revisited), and white background is also the default.